cluster-bootstrap/src/main/scala/akka/management/cluster/bootstrap/ClusterBootstrap.scala

```diff
@@ -64,6 +67,28 @@ final class ClusterBootstrap(implicit system: ExtendedActorSystem) extends Exten
 
   private[this] val _selfContactPointUri: Promise[Uri] = Promise()
 
+
+  // autostart if the extension is loaded through the config extension list
+  private val autostart = system.settings.config.getStringList("akka.extensions").contains(classOf[ClusterBootstrap].getName)
```

Comment:
I'm thinking it should be extensions and added by user in application.conf, having it in library-extensions auto-added by library would be too magic for this IMO

Reply:
yes, I didn't mean to add it to any reference.conf. Just want to be sure that it can be added with `+=` by the operator in `main.conf` which includes `application.conf`

Reply:
and to be more precise
the user may have
```
akka.extensions = [foo]
```

and from main.conf we want to have

```
include application.conf
akka.extensions += [ClusterBootstrap]
```

I think that should work, but config has surprised me previously in this area

Reply:
Hmmm. I'd have to double check that, not sure what Config does when there is a `+=` for a key that is not defined yet.

Reply:
Verified and such an include+append works fine 👍 
```
conf.getStringList("akka.extensions")
val res1: java.util.List[String] = [foo, ClusterBootstrap]
```